### DISCUSSION TITLE
`create-svelte` and SemVer

### DISCUSSION THREAD

**Discussion by merceyz** [sveltejs/kit#4645] [Q&A]
According to the [SemVer specification](https://semver.org/#spec-item-11) `2.0.0` has a higher precedence than `2.0.0-next.129` but according to the [npm registry](https://registry.npmjs.com/create-svelte) `2.0.0` was released at `2020-10-16T22:58:56.546Z` while `2.0.0-next.129` was released at `2022-04-15T12:50:36.481Z`.

Since both `latest` and `next` point to `2.0.0-next.129` I'm assuming this isn't on purpose.
Could `create-svelte@2.0.0` be deprecated to warn users about this or could a new version with a higher precedence be published?

**Comment by benmccann** [comment]
2.0.0-next.132 has been downloaded 3,000+ times this week while 2.0.0 has been downloaded 134. Any idea how those 134 people end up getting that version while everyone else gets the correct version?
